src/Form/BasicSetup.php

```diff
@@ -134,15 +133,15 @@ public function buildForm(array $form, FormStateInterface $form_state, User $use
           $context['setup_context'] = ['plugin_definition' => $plugin_definition];
           $form['#title'] = t('TFA setup - Trusted browsers');
           $setup_plugin = new TfaTrustedBrowserSetup($context);
```

Comment:
Over time, I'm assuming this "setup" plugin will be built into the configuration on top of the plugin systems. Is this correct? I see "trusted browser", but how will this evolve later with the plugin systems?